pages/wysiwyg.html

```diff
@@ -1,6 +1,6 @@
 ---
 title: wysiwyg
-permalink: /wysiwyg/
+permalink: wysiwyg
```

Comment:
You may need to revert this change to have the page resolve correctly in different environments.